use/mobile-gaming-example.md

```diff
@@ -0,0 +1,540 @@
+
+> **Note:** See [LeaderBoard on GitHub](https://github.com/apache/incubator-beam/blob/ee6ad2fe416cd499aa1b6e2a51aa64da0805cc5c/examples/java8/src/main/java/org/apache/beam/examples/complete/game/LeaderBoard.java) for the complete example pipeline program.
+
+### What Does LeaderBoard Do?
+
+The `LeaderBoard` pipeline reads game data published to Pub/Sub in near real-time, and uses that data to perform two separate processing tasks:
+
+* `LeaderBoard` calculates the total score for every unique user and publishes speculative results for every ten minutes of _processing time_. That is, every ten minutes, the pipeline outputs the total score per user that the pipeline has processed to date. This calculation provides a running "leader board" in close to real time, regardless of when the actual game events were generated.
+
+* `LeaderBoard` calculates the team scores for each hour that the pipeline runs. This is useful if we want to, for example, reward the top-scoring team for each hour of play. The team score calculation uses fixed-time windowing to divide the input data into hour-long finite windows based on the _event time_ (indicated by the timestamp) as data arrives in the pipeline.  
+
+    In addition, the team score calculation uses Beam's trigger mechanisms to provide speculative results for each hour (which update every five minutes until the hour is up), and to also capture any late data and add it to the specific hour-long window to which it belongs.
+
+Below, we'll look at both of these tasks in detail.
+
+#### Calculating User Score based on Processing Time
+
+We want our pipeline to output a running total score for each user for every ten minutes that the pipeline runs. This calculation doesn't consider _when_ the actual score was generated by the user's play instance; it simply outputs the sum of all the scores for that user that have arrived in the pipeline to date. Late data gets included in the calculation whenever it happens to arrive in the pipeline as it's running.
+
+Because we want all the data that has arrived in the pipeline every time we update our calculation, we have the pipeline consider all of the user score data in a **single global window**. The single global window is unbounded, but we can specify a kind of temporary cut-off point for each ten-minute calculation by using a processing time [trigger](/learn/programming-guide/#triggers).
+
+When we specify a ten-minute processing time trigger for the single global window, the effect is that the pipeline effectively takes a "snapshot" of the contents of the window every time the trigger fires (which it does at ten-minute intervals). Since we're using a single global window, each snapshot contains all the data collected _to that point in time_. The following diagram shows the effects of using a processing time trigger on the single global window:
```

Comment:
These aren't exact 10 minute intervals -- I believe that's because of the arrival pattern of the data. Nothing will be triggered if there's no data that arrives.

@kennknowles can probably provide more precise phrasing.